cmd/goose/main.go

```diff
@@ -10,12 +10,13 @@ import (
 )
 
 var (
-	flags   = flag.NewFlagSet("goose", flag.ExitOnError)
-	dir     = flags.String("dir", "", "directory with migration files")
-	table   = flags.String("table", "goose_db_version", "migrations table name")
-	verbose = flags.Bool("v", false, "enable verbose mode")
-	help    = flags.Bool("h", false, "print help")
-	version = flags.Bool("version", false, "print version")
+	flags    = flag.NewFlagSet("goose", flag.ExitOnError)
+	dir      = flags.String("dir", ".", "directory with migration files")
+	table    = flags.String("table", "goose_db_version", "migrations table name")
+	verbose  = flags.Bool("v", false, "enable verbose mode")
+	help     = flags.Bool("h", false, "print help")
+	version  = flags.Bool("version", false, "print version")
+	certfile = flags.String("ssl-ca", "", "file path to root CA's certificates in pem format (only support on mysql)")
```

Comment:
Sorry, I meant the CLI arg to be named "certfile" too

Reply:
Oh, I didn't realize your intention. But, the `mysql` command itself has a `--ssl-ca` option and I take the name from it. The `mysql` has a some SSL related options, so I think it seems better to match names.

ref. https://dev.mysql.com/doc/refman/8.0/en/connection-options.html

Reply:
How about it? @VojtechVitek

Reply:
let's use "certfile" please

eventually, we might use this flag for non-MySQL DBs too, ie. Postgres `psql` has
```
--set=sslmode=verify-full --set=sslrootcert=/some-cert.pem
```

Reply:
renamed. Thanks for reviewing.